common/app/assets/assets.scala

```diff
@@ -5,7 +5,7 @@ import org.apache.commons.io.IOUtils
 import play.api.{ Mode, Play }
 import play.api.libs.json.{ JsString, Json, JsObject }
 import conf.Configuration
-import collection.mutable.{ Map => MutableMap }
```

Comment:
Yeah, I wondered if that caused some of our (very seldom) box does not start up problems leading to longer deploys.